src/applications/vaos/tests/new-appointment/va-facility-page-v2-single-site.unit.spec.jsx

```diff
@@ -0,0 +1,867 @@
+import React from 'react';
+import { expect } from 'chai';
+import { Route } from 'react-router-dom';
+
+import { mockFetch, resetFetch } from 'platform/testing/unit/helpers';
+import { fireEvent, waitFor } from '@testing-library/dom';
+import { cleanup } from '@testing-library/react';
+import VAFacilityPage from '../../new-appointment/components/VAFacilityPage/VAFacilityPageV2';
+import {
+  getParentSiteMock,
+  getClinicMock,
+  getVAFacilityMock,
+  getFacilityMock,
+} from '../mocks/v0';
+import {
+  createTestStore,
+  setTypeOfCare,
+  setTypeOfEyeCare,
+  renderWithStoreAndRouter,
+} from '../mocks/setup';
+import {
+  mockEligibilityFetches,
+  mockParentSites,
+  mockSupportedFacilities,
+  mockFacilityFetch,
+} from '../mocks/helpers';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingVSPAppointmentNew: false,
+    vaOnlineSchedulingDirect: true,
+    vaOnlineSchedulingFlatFacilityPage: true,
+  },
+  user: {
+    profile: {
+      facilities: [{ facilityId: '983', isCerner: false }],
+    },
+  },
+};
+
+const parentSite983 = {
+  id: '983',
+  attributes: {
+    ...getParentSiteMock().attributes,
+    institutionCode: '983',
+    authoritativeName: 'Some VA facility',
+    rootStationCode: '983',
+    parentStationCode: '983',
+  },
+};
+
+describe('VAOS integration: VA flat facility page', () => {
```

Comment:
We might be able to combine the multi and single site files in the future, since I don't think there's as big of difference in the code paths for those two scenarios with the new page.